examples/owneredit.py

```diff
@@ -0,0 +1,517 @@
+#!/usr/bin/env python3
+# Impacket - Collection of Python classes for working with network protocols.
+#
+# SECUREAUTH LABS. Copyright (C) 2021 SecureAuth Corporation. All rights reserved.
+#
+# This software is provided under a slightly modified version
+# of the Apache Software License. See the accompanying LICENSE file
+# for more information.
+#
+# Description:
+#   Python script for handling the msDS-AllowedToActOnBehalfOfOtherIdentity property of a target computer
+#
+# Authors:
+#   Charlie BROMBERG (@_nwodtuhs)
+
+import argparse
+import logging
+import sys
+import traceback
+
+import ldap3
+import ssl
+import ldapdomaindump
+from binascii import unhexlify
+from ldap3.protocol.formatters.formatters import format_sid
+
+from impacket import version
+from impacket.examples import logger, utils
+from impacket.ldap import ldaptypes
+from impacket.smbconnection import SMBConnection
+from impacket.spnego import SPNEGO_NegTokenInit, TypesMech
+from ldap3.utils.conv import escape_filter_chars
+from ldap3.protocol.microsoft import security_descriptor_control
+
+
+# Universal SIDs
```

Comment:
move this to a lib, to be used by dacledit.py and this one Also consider fixing single backslash issue in lines 94 to 111. ( see PR #1742 )